nucypher/utilities/prometheus/collector.py

```diff
@@ -99,14 +99,17 @@ def initialize(self, metrics_prefix: str, registry: CollectorRegistry) -> None:
             "learning_status": Enum(f'{metrics_prefix}_node_discovery', 'Learning loop status',
                                     states=['starting', 'running', 'stopped'], registry=registry),
             "known_nodes_gauge": Gauge(f'{metrics_prefix}_known_nodes',
-                                       'Number of currently known nodes',
-                                       registry=registry),
+                                        'Number of currently known nodes',
+                                        registry=registry),
             "work_orders_gauge": Gauge(f'{metrics_prefix}_work_orders',
-                                       'Number of accepted work orders',
-                                       registry=registry),
+                                        'Number of accepted work orders',
+                                        registry=registry),
             "policies_held_gauge": Gauge(f'{metrics_prefix}_policies_held',
-                                         'Policies held',
-                                         registry=registry),
+                                        'Policies held',
+                                        registry=registry),
+            "availability_score_gauge": Gauge(f'{metrics_prefix}_availability_score',
+                                        'Availability score',
+                                        registry=registry),
```

Comment:
See how this block[ looks now](https://github.com/nucypher/nucypher/blob/b52e0a0d83d64e14cf9007df1e10f252b1951eec/nucypher/utilities/prometheus/collector.py#L101-L112)- there were a mix of tabs and spaces in here before

Reply:
💯